### PR TITLE
[#84] Improve Makefile

### LINKED ISSUE #84: Add development targets to Makefile
It would be nice to have some targets in Makefile for development purpose. Currently there is only one way to start app - using `make start` that takes up to 1 minute. It's too much when you just want to update the environment variables. 

Ideally we need hot reload support. At least we should add a target to run the app in development mode.
For instance we can have something like this (probable add condition dependency on target `all`):
```
dev:
	docker run \
	-p $(PORT):3000 \
	-v `pwd`:/usr/src/app \
	-w /usr/src/app node:14-alpine \
	sh -c 'npm start'
```

/cc @alexvanin

### PR DESCRIPTION
**Pull request by KirillovDenis**
close #84 
Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

File: help.mk [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# Show this help prompt
+help:
+	@echo '  Usage:'
+	@echo ''
+	@echo '    make <target>'
+	@echo ''
+	@echo '  Targets:'
+	@echo ''
+	@awk '/^#/{ comment = substr($$0,3) } comment && /^[a-zA-Z][a-zA-Z0-9.%_/-]+ ?:/{ print "   ", $$1, comment }' $(MAKEFILE_LIST) | column -t -s ':' | grep -v 'IGNORE' | sort | uniq
```
Comment (mike-petrov) [inline help.mk:11]: I got error for `make help` or `make`:
```

sh-3.2# make
  Usage:

    make <target>

  Targets:

awk: nonterminated character class ^[a-zA-Z][a-zA-Z0-9.
 source line number 1
 context is
        /^#/{ comment = substr($0,3) } comment && >>>  /^[a-zA-Z][a-zA-Z0-9.%_/ <<< 
```

File: README.md [hunk 2]
```diff
@@ -41,16 +41,30 @@ Set variables in `.env` file
 - `REACT_APP_NEOFS_GAS_TOKEN` - Script hash of the Gas token contract
 
 Run next commands from project's root:
+
```
Comment (masterSplinter01) [inline README.md:44]: Unnecessary empty line?

File: README.md [hunk 3]
```diff
@@ -41,16 +41,30 @@ Set variables in `.env` file
...
+Optionally you can specify `PORT=3000`.
+
 # Make instructions
 * Compile the build using `make` (will be generated in `output` dir)
 * Start app using `make start PORT=3000` (PORT=3000 by default)
 * Clean up cache directories using `make clean`
 * Get release dir with tar.gz using `make release`
 
+Also, you can run all targets in docker (`make docker/*`):
+
```
Comment (masterSplinter01) [inline README.md:63]: Empty line?

File: README.md (hunk 2 above)
Comment (KirillovDenis) [reply README.md:44]: No. It was intended

**Review by mike-petrov — APPROVED** [review]

**Review by masterSplinter01 — DISMISSED** [review]

**Review by mike-petrov — APPROVED** [review]
LGTM

**Pull request merged**